compiler/passes/buildDefaultFunctions.cpp

```diff
@@ -1298,6 +1303,35 @@ static void build_union_assignment_function(AggregateType* ct) {
   normalize(fn);
 }
 
+static void buildClassBorrowMethod(AggregateType* ct) {
+  if (function_exists("borrow", ct, ct))
```

Comment:
The first argument should be the method table, right?
If so, should be dtMethodToken.